bftengine/tests/simpleStorage/FileStorage.cpp

```diff
@@ -140,11 +142,23 @@ void FileStorage::verifyFileMetadataSetup() const {
 void FileStorage::verifyOperation(uint16_t objectId, uint32_t dataLen, const char *buffer) const {
   verifyFileMetadataSetup();
   MetadataObjectInfo *objectInfo = objectsMetadata_->getObjectInfo(objectId);
-  if (!objectInfo || objectId > objectsMetadata_->getObjectsNum() - 1 ||
-      !dataLen || dataLen > objectInfo->maxSize || !buffer) {
-    LOG_FATAL(logger_, "FileStorage::verifyOperation " << WRONG_PARAMETER);
-    throw runtime_error(WRONG_PARAMETER);
+  if (objectInfo && objectId <= (objectsMetadata_->getObjectsNum() - 1) &&
```

Comment:
Nice expansion. It will make tracking down errors easier. I really appreciate this.